src/ResourceManager/Compute/Commands.Compute.Test/ScenarioTests/StrategiesVirtualMachineTests.cs

```diff
@@ -40,6 +40,47 @@ public void TestSimpleNewVm()
 
         [Fact]
         [Trait(Category.AcceptanceType, Category.CheckIn)]
+        public void TestSimpleNewVmWithSystemAssignedIdentity()
+        {
+            ComputeTestController.NewInstance.RunPsTest("Test-SimpleNewVmSystemAssignedIdentity");
+        }
+
+        [Fact]
+        [Trait(Category.AcceptanceType, Category.CheckIn)]
+        public void TestSimpleNewVmWithUserAssignedIdentity()
+        {
+            /**
+             * To record this test run these commands first :
+             * New-AzureRmResourceGroup -Name UAITG123456 -Location 'Central US'
+             * New-AzureRmUserAssignedIdentity -ResourceGroupName  UAITG123456 -NameUAITG123456Identity
```

Comment:
Is there a reason that you didn't include New-AzureRmUserAssignedIdentity in your test?

Reply:
Yes... As of now the UserAssgiedIdentity cmdlets are not GA. This causes issues when you run the tests as the module is not loaded by default.